modules/openapi-generator/src/main/resources/cpp-qt5-client/api-header.mustache

```diff
@@ -1,6 +1,5 @@
 {{>licenseInfo}}
-#ifndef {{prefix}}_{{classname}}_H
-#define {{prefix}}_{{classname}}_H
+#pragma once
```

Comment:
I wonder if the `#pragma once` causes same behaviour in all compilers, when same files with different namespaces in different locations are included. The guard is useful to prevent the compiler from not including it again, because the namespace is different and the C++ linker has to link to the client object from the same namespace.
e.g. when having multiple clients, which is fairly common. One project we are doing has 10+ clients

Reply:
Ok